### PR TITLE
License change due to robot overlords

### PR DESCRIPTION
**Pull request by georgetzavelas**
## Summary of Changes
- Changed the License

**This is bold**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The main robot code for the Merge Robotics (2706) robot for the 2019 FIRST Deep
 
 ## Attribution and license
 
+MY LICENSE!!!
```
Comment (georgetzavelas) [inline README.md]: This license is not specific, can you change it 😢

**Review by georgetzavelas — COMMENTED** [review]
Looks good, see comments

**Comment by kevlam2706** [comment]
Looks OK to me. Did you test this on the robot?

**Comment by georgetzavelas** [comment]
Yeah not sure about this license. Robot overlords may not be happy.

**Review by kevlam2706 — APPROVED** [review]
Everyone loves Shep. I approve!

**Comment by GabbyGenereux** [comment]
Awesome job! 👏👏👏👏

**Comment by jogray72** [comment]
This is super interesting stuff!!!

**Comment by georgetzavelas** [comment]
shufflin-robot

@kevlam2706 Thanks!

**Comment by georgetzavelas** [comment]
Tracking changes in #174

**Pull request merged**